.storybook/theme.ts

```diff
@@ -1,12 +1,14 @@
 
+export const brandImageDark =
+  "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTA4IiBoZWlnaHQ9IjMyIiB2aWV3Qm94PSIwIDAgMTA4IDMyIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgoJPGcgY2xhc3M9InRleHQiPgoJCTxwYXRoIGQ9Ik02OC41MTk5IDIyLjY4NTZWMkg3Mi4yNTQ1VjkuNDg4OTVDNzMuNTE5NCA4LjI4NDYzIDc1LjE0ODggNy42MTY2OSA3Ni44NjkzIDcuNjE2NjlDODAuOTA3MyA3LjYxNjY5IDg0LjQ4OTggMTAuNjkzMiA4NC40ODk4IDE1LjMyODNDODQuNDg5OCAxOS45NjMzIDgwLjkwNzMgMjMuMDUgNzYuODY5MyAyMy4wNUM3NS4wNTc3IDIzLjA1IDczLjM0NzEgMjIuMzMxNCA3Mi4wNzIzIDIxLjAyNTlWMjIuNjg1Nkg2OC41MTk5Wk04MC43NTUzIDE1LjMyODNDODAuNzU1MyAxMi43Mzc1IDc4LjkxMzUgMTAuOTI2IDc2LjUwNDkgMTAuOTI2Qzc0LjE1NzEgMTAuOTI2IDcyLjM0NTYgMTIuNjQ2NCA3Mi4yNTQ1IDE1LjE0NjFWMTUuMzI4M0M3Mi4yNTQ1IDE3LjkxOSA3NC4wOTYyIDE5LjczMDUgNzYuNTA0OSAxOS43MzA1Qzc4LjkxMzUgMTkuNzMwNSA4MC43NTUzIDE3LjkxOSA4MC43NTUzIDE1LjMyODNaTTg2LjIyMDIgMjIuNjg1NlYySDg5Ljk1NDdWMjIuNjg1Nkg4Ni4yMjAyWk05NS4xNzY1IDI4LjA0OTNMOTcuMzUyNSAyMi44NTc3TDkxLjQ0MiA3Ljk4MTAzSDk1LjQ4TDk5LjI0NDcgMTguMzc0NEwxMDMuMDcgNy45ODEwM0gxMDcuMTY5TDk5LjE1MzYgMjguMDU5NEg5NS4xNzY1VjI4LjA0OTNaTTYyLjU4OTYgNy45ODEwM1Y5Ljg1MzI0QzYxLjI5NCA4LjQzNjQyIDU5LjQ4MjQgNy42MjY4NCA1Ny42MjA0IDcuNjI2ODRDNTMuNTgyNiA3LjYyNjg0IDUwIDEwLjcwMzQgNTAgMTUuMzM4NEM1MCAxOS45ODM1IDUzLjU4MjYgMjMuMDUgNTcuNjIwNCAyMy4wNUM1OS41NTM3IDIzLjA1IDYxLjM4NTEgMjIuMjEgNjIuNzEwOSAyMC43MDIxVjIyLjY5NTdINjUuOTY5N1Y3Ljk4MTAzSDYyLjU4OTZaTTYyLjIyNTMgMTUuMzI4M0M2Mi4yMjUzIDE3Ljg4ODcgNjAuMzgzNSAxOS43MzA1IDU3Ljk3NDggMTkuNzMwNUM1NS41NjYxIDE5LjczMDUgNTMuNzI0MiAxNy44ODg3IDUzLjcyNDIgMTUuMzI4M0M1My43MjQyIDEyLjc2NzkgNTUuNTY2MSAxMC45MjYgNTcuOTc0OCAxMC45MjZDNjAuMzIyNiAxMC45MjYgNjIuMTM0MiAxMi42NzY4IDYyLjIyNTMgMTUuMTQ2MVYxNS4zMjgzWiIgZmlsbD0id2hpdGUiIC8+Cgk8L2c+Cgk8ZyBjbGFzcz0idHJpYW5nbGUiPgoJCTxwYXRoIGQ9Ik0xOS4wOTY4IDBMMy4xMTcwMSAyOS4yNDczTDAgMjcuMDYxNEwxNC43ODU2IDBIMTkuMDk2OFpNMTkuMzE5NCAwTDM1LjI5OTIgMjkuMjQ3M0wzOC40MTYyIDI3LjA2MTRMMjMuNjMwNiAwSDE5LjMxOTRaIiBmaWxsPSJ1cmwoI2dyYWRpZW50KSIgLz4KCQk8cGF0aCBkPSJNMzUuMDc2NSAyOS40MTk0TDE5LjIwODEgMTYuOTkxOEwzLjMzOTY2IDI5LjQxOTRMNi41NzgxMyAzMS42ODYzTDE5LjIwODEgMjEuNzk4OUwzMS44MzgxIDMxLjY4NjNMMzUuMDc2NSAyOS40MTk0WiIgZmlsbD0idXJsKCNncmFkaWVudCkiIC8+Cgk8L2c+Cgk8ZGVmcz4KCQk8bGluZWFyR3JhZGllbnQgaWQ9ImdyYWRpZW50IiB4MT0iMTkuMjA4MSIgeTE9IjAiIHgyPSIzIiB5Mj0iMjguNSIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiPgoJCQk8c3RvcCBzdG9wLWNvbG9yPSIjRkYwOTAyIiAvPgoJCQk8c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiNGRjUyMTUiIC8+CgkJPC9saW5lYXJHcmFkaWVudD4KCTwvZGVmcz4KPC9zdmc+Cg==";
+
 export default create({
```

Comment:
My IDE is not happy with the `export default create` way lol. what if we use named export?

```
const theme = create({
  ...
});

export default theme; 
```

Reply:
Sure, why not